core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -166,11 +166,12 @@ private[spark] class CoarseGrainedExecutorBackend(
         exitExecutor(1, "Received LaunchTask command but executor was null")
       } else {
         if (decommissioned) {
-          logError("Asked to launch a task while decommissioned.")
+          val msg = "Asked to launch a task while decommissioned."
+          logError(msg)
```

Comment:
Is there a reason we broke this on two lines (here and elsewhere)? I don't have strong feelings just if theres been a change to the style want to make sure I'm in the loop.

Reply:
It's not broken into two lines for style. I wanted to reuse the "msg" down below. Same deal elsewhere in this PR.